arc_to_parquet/arc_to_parquet.ipynb

```diff
@@ -4,6 +4,27 @@
    "cell_type": "code",
```

Comment:
<p>@yjb-ds redundant cell (will be copied to the function)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>